src/meta/processors/job/JobDescription.h

```diff
@@ -176,7 +176,7 @@ class JobDescription {
   Status status_;
   int64_t startTime_;
   int64_t stopTime_;
-  nebula::cpp2::ErrorCode errCode_{nebula::cpp2::ErrorCode::E_UNKNOWN};
+  nebula::cpp2::ErrorCode errCode_{nebula::cpp2::ErrorCode::E_JOB_PROCESSING};
```

Comment:
The default is `E_JOB_PROCESSING`, which doesn't feel good

Reply:
> The default is `E_JOB_PROCESSING`, which doesn't feel good

Yes,  I know.
Wait for me to think of an awesome name. XDDD

Reply:
What about E_SUBMITTED?

Reply:
What about the display is empty when the job status is `nebula::cpp2::ErrorCode::E_UNKNOWN`？

Reply:
There's some condition job return an unknown error ...